src/views/Resources.vue

```diff
@@ -109,6 +114,38 @@ export default class Resources extends Vue {
 <style lang="less">
 @import "../colors.less";
 
+.create-instance-button {
```

Comment:
Does it somehow differ from the button at types view?
If it's the same style, we need to extract it to avoid duplication

Reply:
I extracted it to the AddButton component